CHANGELOG/CHANGELOG-0.1.md

```diff
@@ -5,3 +5,5 @@ Changes since `v0.1.0`:
 - Fixed number of pending workloads in a BestEffortFIFO ClusterQueue.
 - Fixed bug in a BestEffortFIFO ClusterQueue where a workload might not be
   retried after a transient error.
+- Fixed bug in a BestEffortFIFO ClusterQueue when calling method `DeleteFromQueue`,
```

Comment:
release notes should not contain implementation details.

Maybe something like this:

- Fixed bug in a BestEffortFIFO ClusterQueue where unadmissible workloads were not removed from the ClusterQueue when the corresponding Queue was removed.

Reply:
Agree, let's use the suggested lines

Reply:
Done